HARK/distribution.py

```diff
@@ -821,9 +819,11 @@ class DiscreteDistribution(Distribution):
     ----------
     pmf : np.array
         An array of floats representing a probability mass function.
-    X : np.array or [np.array]
+    X : np.array
         Discrete point values for each probability mass.
-        May be multivariate (list of arrays).
+        For multivariate distributions, the last dimension of X must index
+        "nature" or the random realization. For instance, if X.shape == (2,6,4),
+        the random variable has 4 possible realizations and each of them has shape (2,6).
```

Comment:
I believe it would be more consistent with the other downstream code if the 'nature' dimension was the first dimension, not the last.